src/app/pages/account/account-navigation/account-navigation.component.html

```diff
@@ -6,7 +6,7 @@
         <li
           *ishFeature="item.value.feature || 'always'"
           routerLinkActive="active"
-          [routerLinkActiveOptions]="{ exact: first }"
+          [routerLinkActiveOptions]="{ exact: true }"
```

Comment:
I'm not so sure if this doesn't break the highlight for the "/overview"

Reply:
Not sure what you mean.

Reply:
@shauke can you elaborate as you did the change, please?!

Reply:
Reverted back to `[routerLinkActiveOptions]="{ exact: first }"` as it works nicely with sub routes. 
Unfortunately it is not clear why this is the case since the `exact: first` option is not documented at all?
```
routerLinkActiveOptions: {
    exact: boolean;
}
```
https://angular.io/api/router/RouterLinkActive